content/tracing/search.md

```diff
@@ -0,0 +1,227 @@
+
+* A [**tag**](#tags)
+
+To combine multiple *terms* into a complex query, use any of the following boolean operators:
+
+|              |                                                                                                        |                              |
+| :----        | :----                                                                                                  | :----                        |
+| **Operator** | **Description **                                                                                       | **Example **                 |
+| `AND`        | **Intersection**: both terms are in the selected events (if nothing is added, AND is taken by default) | authentication AND failure   |
+| `OR`         | **Union**: either terms is contained in the selected events                                            | authentication OR password   |
+| `-`          | **Exclusion**: the following term is NOT in the event                                                  | authentication AND -password |
+
+### Facet search
+
+To search on a specific [facet](#facets) you must [add it as a facet first](#create-a-facet) then add `@` to specify you are searching on a facet.
+
+For instance, if your facet name is **url** and you want to filter on the **url** value *www.datadoghq.com* just enter:
+
+`@url:www.datadoghq.com`
+
+### Tags search
+
+Your traces inherit tags from [hosts][4] and [integrations][5] that generate them. They can be used in the search and as facets as well:
+
+| Query                                                          | Match                                                                       |
+| :----                                                          | :---                                                                        |
+| `("env:prod" OR test)`                                         | All traces with the tag `#env:prod` or the tag `#test`                      |
+| `(service:srvA OR service:srvB)` or `(service:(srvA OR srvB))` | All traces that contain tags `#service:srvA` or `#service:srvB`.            |
+| `("env:prod" AND -”version:beta”)`                             | All traces that contain `#env:prod` and that do not contain `#version:beta` |
+
+If your tags don't follow [tags best practices][6] and don't use the `key:value` syntax, use this search query:
+
+* `tags:<MY_TAG>`
+
+### Wildcards
+
+To perform a multi-character wildcard search, use the `*` symbol as follows:
+
+*  `service:web*`  matches every traces that has a service starting by `web`.
+*  `@url:data*`  matches every traces that has a `url` starting by `data`.
+
+### Numerical values
+
+Use `<`,`>`, `<=`, or `>=` to perform a search on numerical attributes. For instance, retrieve all traces that have a response time over 100ms with:
+
+`@http.response_time:>100`
+
+It is also possible to search for numerical attribute within a specific range. For instance, retrieve all your 4xx errors with:
```

Comment:
`It is also possible to search for numerical attributes within a specific range. For instance, retrieve all your 4xx errors with:`